tools/local-env/scripts/start.js

```diff
@@ -9,7 +9,10 @@ if ( process.arch === 'arm64' ) {
 }
 
 // Start the local-env containers.
-execSync( 'docker-compose up -d wordpress-develop', { stdio: 'inherit' } );
+const containers = ( process.env.LOCAL_PHP_MEMCACHED === 'true' )
+	? 'wordpress-develop memcached'
+	: 'wordpress-develop';
+execSync( `docker-compose up -d -- ${containers}`, { stdio: 'inherit' } );
```

Comment:
@johnbillion Is there a specific reason for the inclusion of `--` here? I found that it causes an error in `docker-compose`, at least in v1.25.0 on Linux:

```
npm run env:start

> WordPress@6.1.0 env:start /home/westonruter/repos/wordpress-develop
> node ./tools/local-env/scripts/start.js

Creating network "wordpress-develop_wpdevnet" with driver "bridge"
ERROR: No such service: --
child_process.js:866
    throw err;
    ^

Error: Command failed: docker-compose up -d -- wordpress-develop
    at checkExecSyncError (child_process.js:790:11)
    at execSync (child_process.js:863:15)
    at Object.<anonymous> (/home/westonruter/repos/wordpress-develop/tools/local-env/scripts/start.js:11:1)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:75:12)
    at internal/main/run_main_module.js:17:47 {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 8626,
  stdout: null,
  stderr: null
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! WordPress@6.1.0 env:start: `node ./tools/local-env/scripts/start.js`
npm ERR! Exit status 1
```

If I remove the `--` then the command runs successfully.

Reply:
I assume there was a reason but I don't remember it. Can you open a ticket? 👍

Reply:
Here you go: https://core.trac.wordpress.org/ticket/56550